js/samples/04.ai-apps/d.assistants-mathBot/infra/azure.bicep

```diff
@@ -67,9 +67,17 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
           value: botAadAppClientSecret
         }
         {
-          name: 'OPENAI_KEY'
+          name: 'AZURE_OPENAI_KEY'
           value: openAIKey
```

Comment:
update `value` to `azureOpenAIKey` (same for the endpoint value to `azureOpenAIEndpoint`)

Reply:
good catch

Reply:
fixed